content/docs/2.15/concepts/scaling-jobs.md

```diff
@@ -16,250 +18,33 @@ For example, if you wanted to use KEDA to run a job for each message that lands
 1. As additional messages arrive, additional jobs are created.  Each job processes a single message to completion.
 1. Periodically remove completed/failed job by the `SuccessfulJobsHistoryLimit` and `FailedJobsHistoryLimit.`
 
```

Comment:
Same here about spec section

Reply:
I meant same as https://github.com/kedacore/keda-docs/pull/1389#discussion_r1630984363, sorry.

Reply:
Got it. See comment [there](https://github.com/kedacore/keda-docs/pull/1389#discussion_r1630984363).